spec/system/user_viewing_privacy_statement_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.feature 'Viewing privacy policy' do
+  scenario 'User views the privacy policy' do
+    given_i_am_on_the_start_page
+    when_i_can_click_on_privacy_policy
+    then_i_can_see_the_privacy_policy
+  end
+
+  def given_i_am_on_the_start_page
+    visit '/'
+  end
+
+  def when_i_can_click_on_privacy_policy
+    within('.govuk-footer') { click_link t('layout.privacy_policy') }
+  end
+
+  def then_i_can_see_the_privacy_policy
+    expect(page).to have_content(t('page_titles.privacy_policy'))
```

Comment:
```suggestion
    expect(page).to have_content(t('page_titles. privacy_candidate'))
```

This should fix the build :)

Reply:
Eek. I’m testing blind as wasn’t able to get tests running locally last week. Will try again for my next PR :-)